samples/v1.5/Test/TextBlock.Markdown.NumberStart.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+  "type": "AdaptiveCard",
+  "version": "1.3",
+  "body": [
+	{
+	  "type": "TextBlock",
+	  "text": "1. First item in the list;"
+	},
+	{
+	  "type": "TextBlock",
+	  "text": "2. Second item in the list;"
+	},
+	{
+	  "type": "TextBlock",
+	  "text": "3. Third item in the list;"
```

Comment:
we should have a test to make sure the second and third items in a markdown list are incremented correctly:

```json
{
    "type": "TextBlock",
    "text": "10. the tenth thing\r10. the list is still going!\r10. should be 12!"
}
```